examples/FaxFiles.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+require_once __DIR__ . "/vendor/autoload.php";
+
+$config = Dropbox\Sign\Configuration::getDefaultConfiguration();
+
+// Configure HTTP basic authorization: api_key
+$config->setUsername("YOUR_API_KEY");
+
+$faxApi = new Dropbox\Sign\Api\FaxApi($config);
+
+$faxId = "fa5c8a0b0f492d768749333ad6fcc214c111e967";
+
+try {
+    $faxApi->getFaxFiles($faxId);
```

Comment:
```suggestion
    $result = $faxApi->getFaxFiles($faxId);
    copy($result->getRealPath(), __DIR__ . '/file_response.pdf');
```

`FaxApi::getFaxFiles` to be renamed to `faxFiles`